ibazel/live_reload/server.go

```diff
@@ -82,22 +65,50 @@ func (l *LiveReloadServer) TargetDecider(rule *blaze_query.Rule) {
 		}
 	}
 }
-func contains(l []string, e string) bool {
-	for _, i := range l {
-		if i == e {
-			return true
-		}
-	}
-	return false
+
+func (l *LiveReloadServer) ChangeDetected(targets []string, changeType string, change string) {
 }
 
-func (l *LiveReloadServer) ChangeDetected(changeType string) {}
+func (l *LiveReloadServer) BeforeCommand(targets []string, command string) {}
 
-func (l *LiveReloadServer) BeforeCommand(command string) {}
-func (l *LiveReloadServer) AfterCommand(command string, success bool) {
+func (l *LiveReloadServer) AfterCommand(targets []string, command string, success bool) {
 	if l.lrserver != nil {
 		fmt.Fprintf(os.Stderr, "Triggering live reload\n")
 		l.lrserver.Reload("reload")
+		l.reloadTriggered(targets)
```

Comment:
I think I would move `l.lrserver.Reload("reload")` into `l.reloadTriggered` to ensure you we never accidentally call reload without notifying the profiler

Reply:
👍